arenadata/Dockerfile.ubuntu

```diff
@@ -24,10 +24,13 @@ RUN set -eux; \
 # Alter precedence in favor of IPv4 during resolving
     echo 'precedence ::ffff:0:0/96  100' >> /etc/gai.conf; \
 # Packages for tests
-    DEBIAN_FRONTEND=noninteractive apt install -y krb5-kdc krb5-admin-server fakeroot sudo python-pip; \
+    DEBIAN_FRONTEND=noninteractive apt install -y krb5-kdc krb5-admin-server fakeroot sudo python-pip openjdk-11-jdk; \
 # Install allure-behave for behave tests
     pip2 install allure-behave==2.4.0; \
-    pip2 cache purge
+    pip2 cache purge; \
+    echo '#!/bin/sh' >> /etc/profile.d/jdk_home.sh; \
```

Comment:
Can we use one command to create `/etc/profile.d/jdk_home.sh` instead of three?
Can we replace `>>` with `>`?

Reply:
I only could combine it into one line

Reply:
reverted and use one `>` at first place